src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -151,6 +152,54 @@ const WorkspaceAccessError = () => {
   ])
 }
 
+const checkBucketAccess = withErrorReporting('Error checking bucket access', async (signal, namespace, name) => {
+  try {
+    await Ajax(signal).Workspaces.workspace(namespace, name).checkBucketReadAccess()
+    return true
+  } catch (error) {
+    if (error.status === 403) {
+      notify('error', div([
```

Comment:
The link to the Google Bucket should be disabled if a user is getting this error. That way, even if they dismiss the error, they still cannot try to access the Bucket.

<img width="403" alt="Screen Shot 2019-05-09 at 9 34 22 AM" src="https://user-images.githubusercontent.com/2395211/57457518-aea2ff00-723d-11e9-9ae8-655954301f87.png">